src/main/java/org/broadinstitute/consent/http/resources/LibraryCardResource.java

```diff
@@ -118,10 +119,15 @@ public Response updateLibraryCard(@Auth AuthUser authUser, @PathParam("id") Inte
   public Response deleteLibraryCard(@Auth AuthUser authUser, @PathParam("id") Integer id) {
     User user = userService.findUserByEmail(authUser.getEmail());
     LibraryCard card = libraryCardService.findLibraryCardById(id);
-    User lcUser = userService.findUserById(card.getUserId());
+    User lcUser = null;
+    try {
+      lcUser = userService.findUserById(card.getUserId());
+    } catch (NotFoundException nfe) {
+      // LC User can be null - do not need to error here
+    }
     try {
       // If user is not an admin and SO institutionID doesn't match the user's throw an exception
-      if (!checkIsAdmin(user) && !lcUser.getInstitutionId().equals(user.getInstitutionId())) {
+      if (lcUser != null && !checkIsAdmin(user) && !lcUser.getInstitutionId().equals(user.getInstitutionId())) {
```

Comment:
In other pull requests, we've started to use the Institution object's built-in equals comparison so that we can hold all of the business logic there.  It largely does the same thing as you're doing here right now.

```suggestion
      if (lcUser != null && !checkIsAdmin(user) && !lcUser.getInstitution().equals(user.getInstitution())) {
```